cmd/livepeer/livepeer.go

```diff
@@ -254,14 +254,11 @@ func main() {
 
 		//Get the Eth client connection information
 		gethUrl := ""
-		if *ethIpcPath != "" {
-			//Connect to specified IPC file
-			gethUrl = *ethIpcPath
-		} else if *ethUrl != "" {
-			//Connect to specified websocket
+		if *ethUrl != "" {
```

Comment:
Could we simplify this if/else into:

```
if *ethUrl == "" {
    glog.Error("Need to specify ethUrl")
    return
}
```

since `*ethUrl` would either be equal to the user specified value or one of the network defaults at this point? We could also remove the extra `gethUrl` variable.

Reply:
Fixed